src/plugins/plot/spectralAggregatePlot/SpectralAggregateView.vue

```diff
@@ -21,15 +21,314 @@
 -->
 
 <template>
-<div>
-
+<div class="c-spectral-aggregate-plot-view gl-plot plot-legend-bottom"
+     :class="{'plot-legend-expanded': legendExpanded, 'plot-legend-collapsed': !legendExpanded }"
```

Comment:
We don't need the legend. We should remove the code related to this functionality.